apps/server/src/modules/tool/controller/tool-school.controller.ts

```diff
@@ -67,6 +68,28 @@ export class ToolSchoolController {
 		return mapped;
 	}
 
+	@Put('/:schoolExternalToolId')
+	@ApiOkResponse({ description: 'The Tool has been successfully updated.', type: SchoolExternalToolResponse })
+	@ApiForbiddenResponse()
+	@ApiUnauthorizedResponse()
+	@ApiResponse({ status: 400, type: ValidationError, description: 'Request data has invalid format.' })
```

Comment:
Why don't you use @ApiBadRequestResponse here? Because above you used the shorthands.

Reply:
Yes you are right. You can use that too